DetoxSync/DetoxSync/ReactNativeSupport/DTXReactNativeSupport.m

```diff
@@ -215,25 +226,30 @@ + (void)waitForReactNativeLoadWithCompletionHandler:(void (^)(void))handler
   __block __weak id observer;
   __block __weak id observer2;
 
-  observer = [[NSNotificationCenter defaultCenter] addObserverForName:@"RCTJavaScriptDidLoadNotification" object:nil queue:nil usingBlock:^(NSNotification * _Nonnull note) {
```

Comment:
Don't see any reason to wait for RCTJavaScriptDidLoadNotification when in any case we will need to wait for RCTContentDidAppearNotification. Also RCTJavaScriptDidLoadNotification is not emitted on RN 0.76 with New Arch.